src/options.rs

```diff
@@ -85,6 +91,34 @@ impl ColumnOptions {
 		}
 		true
 	}
+
+	fn from_string(s: &str) -> Option<Self> {
```

Comment:
This function is not ideal to write/maintain, but having a json lib deps does not seems good either.
Maybe switching simple .properties file at some point could be good.